### ISSUE TITLE
MapGL's children are not created in `componentDidMout` call 

### ISSUE THREAD

**Issue by xintongxia** [visgl/react-map-gl#1098]
Issue:

Children of MapGL are not created when `componentDidMout`.
```
import MapGL from 'react-map-gl';

componentDidMount() {
  console.log(this._divRef); // null
}

render() {
  return (
    <MapGL>
    <div
      ref={_ => this._divRef = _}
    />
  </MapGL>
 )
}

```

After some debugging, found out this is because of [`AutoSizer`](https://github.com/visgl/react-map-gl/blob/master/src/components/static-map.js#L310) used in `StaticMap` component. However the `AutoSizer` in `react-virtualized` by the same Author doesn't have this issue. Apparently `react-virtualized` bundle size is much bigger than 'react-virtualized-auto-sizer', but 'react-virtualized-auto-sizer' seems not be maintained any more.

Sandbox:
https://codesandbox.io/s/debug-autosizer-h0dp4

Related Issue:
https://github.com/uber/nebula.gl/issues/294

**Comment by Pessimistress** [comment]
Should this be an issue in the `react-virtualized-auto-sizer` repo?

**Comment by Pessimistress** [comment]
Addressed in #1174
